strapi/design-system
# [chore] Add connecting to Strapi section to CONTRIBUTING

### What does it do?

Adds a section describing how the design system can be connected to Strapi for testing in the CMS

### Why is it needed?

Testing in isolation using Storybook is great but you might also want to test your work in the CMS

### How to test it?

Make a change somewhere in the design system, then follow the steps to link the design system to your Strapi application. Verify the local version of the DS is being used.

--- 

More or less the same as https://github.com/joshuaellis/strapi-design-system#connecting-to-strapi
